test/tools/NamedPipeConnection/src/code/Microsoft.PowerShell.NamedPipeConnection.csproj

```diff
@@ -15,8 +15,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
-    <Reference Include="System.Management.Automation">
-      <HintPath>.\Ref\System.Management.Automation.dll</HintPath>
-    </Reference>
+    <PackageReference Include="System.Management.Automation" Version="7.4.0-preview.1" />
```

Comment:
Should this be automated in some way?

Reply:
Good point. I will add this csproj to the dependabot monitored csproj files